ocs_ci/helpers/helpers.py

```diff
@@ -716,6 +716,76 @@ def create_multiple_pvcs(
     return ocs_objs
 
 
+def create_bulk_pvcs(
+    sc_name,
+    namespace,
+    number_of_pvc=1,
+    size=None,
+    access_mode=constants.ACCESS_MODE_RWO,
+):
+    """
+    Create one or more PVC as a bulk or one by one
+    Args:
+        sc_name (str): The name of the storage class to provision the PVCs from
+        namespace (str): The namespace for the PVCs creation
+        number_of_pvc (int): Number of PVCs to be created
+        size (str): The size of the PVCs to create
+        access_mode (str): The kind of access mode for PVC
+
+    Returns:
+         list: List of PVC objects and tmp dir where the yamls are created
+    """
+
+    pvc_data = templating.load_yaml(constants.CSI_PVC_YAML)
+    pvc_data["metadata"]["namespace"] = namespace
+    pvc_data["spec"]["accessModes"] = [access_mode]
+    pvc_data["spec"]["storageClassName"] = sc_name
+    if size:
+        pvc_data["spec"]["resources"]["requests"]["storage"] = size
+    if access_mode == "ReadWriteMany" and "rbd" in sc_name:
+        pvc_data["spec"]["volumeMode"] = "Block"
+    else:
+        pvc_data["spec"]["volumeMode"] = None
+
+    # Creating tem directory to hold the files for the PVC creation
+    tmpdir = tempfile.mkdtemp()
+    logger.info("Creating the PVC yaml files for creation in bulk")
+    ocs_objs = []
+    for _ in range(number_of_pvc):
+        name = create_unique_resource_name("test", "pvc")
+        logger.info(f"Adding PVC with name {name}")
+        pvc_data["metadata"]["name"] = name
+        templating.dump_data_to_temp_yaml(pvc_data, f"{tmpdir}/{name}.yaml")
+        ocs_objs.append(pvc.PVC(**pvc_data))
+
+    logger.info("Creating all PVCs as bulk")
+    oc = OCP(kind="pod", namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    cmd = f"create -f {tmpdir}/"
+    oc.exec_oc_cmd(command=cmd, out_yaml_format=False)
+
+    # Letting the system 1 sec for each PVC to create.
+    # this will prevent any other command from running in the system in this
+    # period of time.
+    logger.info(
+        f"Going to sleep for {number_of_pvc} sec. "
+        "until starting verify that PVCs was created."
+    )
+    time.sleep(number_of_pvc)
+
+    return ocs_objs, tmpdir
+
+
+def delete_bulk_pvcs(pvc_yaml_dir):
+    """
+    Deletes all the pvcs created from yaml file in a provided dir
+    Args:
+        pvc_yaml_dir (str): Directory in which yaml file resides
+    """
+    oc = OCP(kind="pod", namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    cmd = f"delete -f {pvc_yaml_dir}/"
+    oc.exec_oc_cmd(command=cmd, out_yaml_format=False)
+
```

Comment:
you need to wait and verify that all PVC's was deleted.

Reply:
will add time.sleep (100). Or more ?

Reply:
I've added sleep (500) and calidation of pv deletion. Will execute it to check whether this timeout is enough.

Reply:
@ypersky1980 we should also check if the total rbd images in ceph system match with the PV count at OCS. This will ensure there are no leftover rbd images from the backend after deletion.

Reply:
@ypersky1980 have you tried kube_job for creation ?, if so deletion is just one line, i.e. `kube_job.delete(namespace)` which kind of does the same.
Eg: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pvc_creation_deletion_scale.py#L138

Reply:
@ramkiperiy  no, I did not try this, I thought that it would be a good idea to implement bulk deletion in a similar way the bulk creation is implemented.  The current implementation is also rather short and the separate function can be useful as well for the future. But thank you for pointing this out, I'll keep it in mind for future.